src/components/quiz-questions/EquilibriumQuestion.tsx

```diff
@@ -63,10 +72,12 @@ const EquilibriumQuestion: React.FC = () => {
     );
     // Hide the question if the user has already answered correctly, and moved on
     // to the next page
-    const hide = page > PROMPT_TO_ADJUST_B && formState === FormState.Correct;
+    const hide =
+        page > startPage.current?.page && formState === FormState.Correct;
```

Comment:
`startPage.page` seems like it has multiple responsibilities which is a little confusing. Consider splitting out the role it plays in checking for initialization by adding some kind of `isInitialized` state/ref flag, and then `const hide = !isInitialized && page > startPage.current.page && ...`?

Reply:
rename `startPage` to `firstVisiblePage`?